index.html

```diff
@@ -83,6 +84,9 @@ <h1 class="headline-title">
       </a>
     </section>
 
+    <!-- The Modal -->
+    <section id="myModal" class="modal"></section>
+
     <!-- work section -->
     <section id="portfolio" class="work-container container">
       <div class="work-heading">
```

Comment:
- [ ] Kindly make sure the **work section** is created dynamically with the information from the projects array(all the HTML of the section is created when the page loads). You should remove all the created HTML code for the project's list, and instead, it should be added via JavaScript. Here is an **[example](https://github.com/olawale-o/olawale-o.me/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)** for more details.

- [ ] Once you implement the above changes in your `.js` file you will make sure that your **Modal popup** data is populated with the data from the projects array you created previously.